### PR TITLE
Add condition to either use NATS or RabbitMQ.

### PR DESCRIPTION
**Pull request by SaraGhlm**
Added queue system and rabbitmq parameters to values.yaml.
Added condition to use one of the dependencies (NATS or RabbitMQ).
Also, added RabbitMQ parameters to places that were using NATS.

### CODE REVIEW

**Review by emadatti — APPROVED** [review]

File: charts/mainflux/values.yaml [hunk 1]
```diff
@@ -11,6 +11,21 @@ defaults:
   replicaCount: 3
   natsPort: 4222
   jaegerPort: 6831
+  rabbitmqPort: 5672
+
+queueSystem:
+  natsQueue: true
+  rabbitmqQueue: false
```
Comment (HolyKau) [inline charts/mainflux/values.yaml:18]: What happens if both of these are set to true?

File: charts/mainflux/values.yaml [hunk 2]
```diff
@@ -11,6 +11,21 @@ defaults:
...
+
+rabbitmq:
+  tlsConfig: ""
+  durable: true
+  ttl: 345600
+  priority: 5
+  contentType: "application/json"
+  subSystem: "smc"
+  severity: 6
+  queue: "topic1:replyTo1,topic2:replyTo2"
```
Comment (HolyKau) [inline charts/mainflux/values.yaml]: Optional settings should be commented out.

File: charts/mainflux/values.yaml (hunk 1 above)
Comment (SaraGhlm) [reply charts/mainflux/values.yaml:18]: In this case, we would have both of them running. We need to set only one to true at each time.

Comment (SaraGhlm) [reply charts/mainflux/values.yaml:18]: Also, we cannot use only one boolean, as in the condition statement in Chart.yaml we cannot use the negation of a condition.

**Pull request merged**